src/cli/peakdetector/PeakDetectorCLI.cpp

```diff
@@ -585,7 +608,7 @@ string PeakDetectorCLI::cleanSampleName(string sampleName) {
         return out.toStdString();
 }
 
-void PeakDetectorCLI::writeReport(string setName) {
+void PeakDetectorCLI::writeReport(string setName,QString jsPath,QString nodePath) {
```

Comment:
@Ky5595 You can't pass these arguments here. Not related to writeReport function.

Reply:
@sahil21  For this sprint, we can pass these as it is, it will require significant changes in the code. 
In the next sprint, we are going to remove the node_modules from repo in any case.